.github/workflows/regen_certs.yml

```diff
@@ -0,0 +1,34 @@
+name: Regenerate WPT certificates
+on:
+  schedule:
+    # Our certificates are good for a few years, but we regenerate them every 6
+    # months to be sure.
+    - cron: '0 0 1 */6 *'
```

Comment:
I'm not sure how often we want to run this. Running it only every 6 months means there's a good chance the script will break before its run again, but running it more frequently is sort of useless churn. Thoughts welcome.

Reply:
Shall we do a daily dry run and update the cert only monthly (or biannually)?

Reply:
I think updating the certs monthly ensures that everyone has processes that can cope with the certs being updated monthly. Updating twice a year seems more risky since people could handle breakage in some way that doesn't generalise.

Reply:
Done; running every month with `--force`.